### ISSUE TITLE
Disable checking if there is no `cspell.json` config file

### ISSUE THREAD

**Issue by WoodyWoodsta** [streetsidesoftware/vscode-spell-checker#2152]
I'm not sure if I'm missing it, or it really doesn't exist, but I'd really like to be able to disable the plugin on VSCode windows (be that a folder or workspace) if there is no `cspell.json` file. Some projects I work on use the checker, while others don't, and I don't want spell checking on the latter.

Otherwise, thanks for a great tool!

**Comment by Jason3S** [comment]
@WoodyWoodsta,

I think the best way to handle this is to add a `cspell.json` file one level above the project directory to turn off the spell checker.

**`cspell.json`**
```js
{ "enable": false }
```

Example directory structure:

```
~
\-- projects
    |-- cspell.json - { enable: false }
    \-- project 1
        |-- cspell.json 
    \-- project 2
    ...
```

**Comment by WoodyWoodsta** [comment]
Thanks @Jason3S - will give this a try. Is it restricted to 1 level above, or could I place this at the root of my home folder to apply to all projects?

**Comment by Jason3S** [comment]
@WoodyWoodsta Let me double check. That is how it was intended.

**Comment by WoodyWoodsta** [comment]
@Jason3S I've actually just given it a try. It picks up the file correctly, but the `enable: false` does not appear to disable the spell checking.

**Comment by Jason3S** [comment]
@WoodyWoodsta,

It is: `{ "enabled": false }` not `enable`

**Comment by Jason3S** [comment]
But the extension incorrectly reports that it is checked, but if you make a spelling error, it won't get reported.

**Comment by WoodyWoodsta** [comment]
Changed to `enabled` after your edit and it works perfectly now. I'd say I'm happy with that! Feel free to close this issue if you think this is as far as the extension will go!
